JOSS/paper.bib

```diff
@@ -0,0 +1,80 @@
+@article{bezanson2017julia,
+  title={Julia: A fresh approach to numerical computing},
+  author={Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B},
+  journal={SIAM review},
+  volume={59},
+  number={1},
+  pages={65--98},
+  year={2017},
+  publisher={SIAM},
+  url={https://doi.org/10.1137/141000671}
```

Comment:
JOSS likes DOI to always be included where possible
```suggestion
  doi={10.1137/141000671}
```

URL can be generated from DOI trivial by referenencing software.
(JabRef at least does it, i assume the expensive ones do also)